pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,192 @@
+# Deploying to Kubernetes
+
+This tutorial teaches deploying to Kubernetes. You'll need to prepare your
+agents with the required packages, cluster access, and Docker registry access
+before hand.
```

Comment:
It would be good to have this a bit friendlier and mention something about the deploy process. Something about how we're using the more complex deploy pipeline way of doing deploys? Then we could link to the overview for more info about deploys.